src/AppBundle/Resources/views/Report/MoneyIn/step.html.twig

```diff
@@ -7,15 +7,24 @@
 
 {% set clientTransOptions = {'%client%': report.client.firstname | e } %}
 {% set groupString = ('form.group.entries.' ~ transaction.group) | trans(clientTransOptions) %}
+{% set category = transaction.category %}
 {% set categoryString = ('form.category.entries.' ~ transaction.category ~ '.label') | trans(clientTransOptions) %}
 {% set transOptions = clientTransOptions | merge({'%group%': groupString, '%category%': categoryString}) %}
 
 {% block htmlTitle %}{{ 'stepPage.moneyIn.htmlTitle' | trans(transOptions) }}{% endblock %}
 {% block pageTitle %}
-    {% if step == 1 %}
-        {{ 'stepPage.moneyIn.pageSectionTitle' | trans(transOptions) }}
-    {% else %}
-        {{ 'stepPage.moneyIn.pageSectionTitleWithCategory' | trans(transOptions) }}
+    {{ 'stepPage.moneyIn.pageTitle' | trans(transOptions) }}{% if step != 1 %}: {% endif %}
+    {% if step == 2 %}{{ groupString | lower }}{% endif %}
+    {% if step == 3 %}
+        {% if transaction.group == 'state-benefits' %}
+            {% if categoryString == 'Other benefits' %}
```

Comment:
@Biggsen  this one relies on the translation key. Better to rely on the category ID, that never changes